### PR TITLE
Prepare v2.0.4

### PR DESCRIPTION
**Pull request by jrick**
Updates dcrd to release-v2.0.4 tag.

Updates dcrwallet to release-v2.0.4 tag.

Updates dcrctl to release-v2.0.4 tag.

Updates politeia to v1.5.0 tag.

Updates dcrlnd to v0.7.4 tag.

Renames dexc distribution to bisonwallet and updates to v1.0.0.

Updates decred-release to master branch.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -87,10 +87,10 @@ type manifestLine struct {
 type manifest []manifestLine
 
 const (
-	decredRelver = "v2.0.3"
-	dexcRelver   = "v0.6.3"
-	ldVersion    = "2.0.3"
-	dexcLdVer    = "0.6.3"
+	decredRelver = "v2.0.4"
+	dexcRelver   = "v1.0.0"
```
Comment (davecgh) [inline main.go:91]: bisonwRelver?

File: main.go [hunk 2]
```diff
@@ -87,10 +87,10 @@ type manifestLine struct {
...
+	ldVersion    = "2.0.4"
+	dexcLdVer    = "1.0.0"
```
Comment (davecgh) [inline main.go:93]: bisonwLdVer?

**Review by davecgh — COMMENTED** [review]
Also need to update `showvers.sh` for the `dexc`, `dexcctl`, and `dexc-tray` renames.

File: main.go (hunk 1 above)
Comment (jrick) [reply main.go:91]: was trying to keep the diff minimal

**Comment by jrick** [comment]
does showvers.sh even work properly on windows? it's not adding the .exe extensions

**Comment by davecgh** [comment]
> does showvers.sh even work properly on windows? it's not adding the .exe extensions

It works fine under msys2 at least.  I used it to paste the details every release.

**Review by alexlyp — APPROVED** [review]

**Review by davecgh — APPROVED** [review]
```
windows: dist "dcrinstall-manifests" manifest hash: SHA256:a66949d9bcae032413c5e224132ca9df1a93c33ba6cfdc6104e3360d8e9a517c (go version go1.23.0 windows/amd64)
  linux: dist "dcrinstall-manifests" manifest hash: SHA256:a66949d9bcae032413c5e224132ca9df1a93c33ba6cfdc6104e3360d8e9a517c (go version go1.23.0 linux/amd64)
```

Windows versions:

```
$ ./showvers.sh
dcrd version 2.0.4+release (Go version go1.23.0 windows/amd64)
dcrctl version 2.0.4+release (Go version go1.23.0 windows/amd64)
dcrwallet version 2.0.4+release (Go version go1.23.0 windows/amd64)
bisonw version 1.0.0+release (Go version go1.23.0 windows/amd64)
bisonw version 1.0.0+release (Go version go1.23.0 windows/amd64)
bwctl version 1.0.0+release (Go version go1.23.0 windows/amd64)
bwctl required RPC server version: 0.3.0
dcrlncli version 0.7.4+release (Go version go1.23.0 windows/amd64)
dcrlnd version 0.7.4+release (Go version go1.23.0 windows/amd64)
politeiavoter version 1.5.0+release (Go version go1.23.0 windows/amd64)
```

Linux versions:

```
$ ./showvers.sh
dcrd version 2.0.4+release (Go version go1.23.0 linux/amd64)
dcrctl version 2.0.4+release (Go version go1.23.0 linux/amd64)
dcrwallet version 2.0.4+release (Go version go1.23.0 linux/amd64)
bisonw version 1.0.0+release (Go version go1.23.0 linux/amd64)
bwctl version 1.0.0+release (Go version go1.23.0 linux/amd64)
bwctl required RPC server version: 0.3.0
dcrlncli version 0.7.4+release (Go version go1.23.0 linux/amd64)
dcrlnd version 0.7.4+release (Go version go1.23.0 linux/amd64)
politeiavoter version 1.5.0+release (Go version go1.23.0 linux/amd64)
```

**Pull request merged**